backend/app/api/endpoints/knowledge.py

```
@@ -686,3 +689,78 @@ def batch_disable_documents(
+            db=db,
+            start_time=start_time,
+            end_time=end_time,
+            user_id=effective_user_id,
+            page=page,
+            page_size=page_size,
+        )
+    except ValueError as e:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail=str(e),
+        )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)